.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+/**
+ * Create comment on pull request
+ *
+ * @param {Number} number - The pull request number
+ * @param {String} messageBody - The comment message
+ * @param {Object} octokitClient - The ocktokit client
+ * @returns {Promise}
+ */
+async function createComment(number, messageBody, octokitClient) {
```

Comment:
Sadly Expensify doesn't use `async/await` in any of our Javascript code. There is a good explanation of this in our [style guide](https://github.com/Expensify/Expensify.cash/blob/master/STYLE.md#esnext-are-we-allowed-to-use-insert-new-language-feature-why-or-why-not). So you'll need to convert all your code to promises instead. Sorry 🙃